src/audio/base_fw.c

```diff
@@ -394,6 +463,10 @@ static int basefw_set_large_config(struct comp_dev *dev,
 						last_block, data_offset, data);
 	case IPC4_ENABLE_LOGS:
 		return ipc4_logging_enable_logs(first_block, last_block, data_offset, data);
+	case IPC4_REGISTER_KCPS:
```

Comment:
@kfrydryx did you try to clarify with ACE FW/Windows SW why do we have two IPC's doing actually the same?